### PR TITLE
Feature/api result integration

### PR DESCRIPTION
**Pull request by tam-nguyen-senseye**
This PR contains updates to the SDK in response to a recent overhaul of Senseye's public API.

This overhaul included changes to the authentication method, available compute endpoints, and expected request/response data formats.

### CODE REVIEW

File: src/components/surveys/Validation.tsx [hunk 1]
```diff
@@ -14,61 +14,50 @@ import {
...
       setResultIcon(Xmark);
       setResultMsg(
-        'Senseye models indicate the user is too fatigued for duty.'
+        'Senseye models indicate the user is likely too fatigued or intoxicated for duty.'
       );
-    } else if (
-      result.prediction.predicted_state ===
-      Constants.PredictedState.NOT_READY_BAC
-    ) {
+    } else if (result.prediction.state === Constants.PredictedState.UNKNOWN) {
```
Comment (Haplo-Dragon) [inline src/components/surveys/Validation.tsx:57]: Would a `switch...case` statement work better here, or does that make React angry?

**Review by Haplo-Dragon — APPROVED** [review]
You could swap in a switch statement if you wanted. Other than that, looks good to me!

File: src/components/surveys/Validation.tsx (hunk 1 above)
Comment (tam-nguyen-senseye) [reply src/components/surveys/Validation.tsx:57]: that might be a good idea. i'll take a look

**Comment by tam-nguyen-senseye** [comment]
https://app.clubhouse.io/senseye/story/17530/update-sdk-to-speak-to-api

**Pull request merged**